app.js

```diff
@@ -1,61 +1,57 @@
-var express = require('express');
-var app = express();
-var redis = require("redis");
-var redisOptions = {
-  host:'ec2-52-27-152-61.us-west-2.compute.amazonaws.com',
-  port:6379,
-  total_retry_time:300000
-};
-var client = redis.createClient(redisOptions);
-var refreshCache = require('./refreshCache');
-
-var log4js = require('log4js');
-log4js.configure({
-  'appenders': [
-    {
-      type      : 'console',
-      category  : 'purdue-laundry'
-    },
-    {
-      type      : 'loggly',
-      token     : '5ac059d3-1b8c-4e5e-9059-0b6b080b3fee',
-      subdomain : 'purduesigapp',
-      tags      : ['purdue-laundry'],
-      category  : 'purdue-laundry',
-      json      : true
-    },
-  ]
-});
+// Initialize Express
+const express = require('express');
+const app = express();
+app.set('port', (process.env.PORT || 5000));
 
-var logger = log4js.getLogger('purdue-laundry');
+// Initialize cache
+const redis = require('redis');
+const redisOptions = {
+	host: 'redis',
+	port: 6379,
+	total_retry_time: 300000
+};
+const client = redis.createClient(redisOptions);
 
-client.on("error",function(err){
-  logger.error('redis error - ' + err);
+client.on('error', err => logger.error('redis error - ' + err));
+client.on('connect', () => {
+	logger.info('redis connected');
+	app.listen(app.get('port'), logger.info('Application listening on port', app.get('port')));
 });
+const Redis = require('./server/classes/Redis');
+let r = new Redis(client);
 
-client.on('connect', function(connect){
-  logger.info('redis connected');
-  var server = app.listen(app.get('port'), function () {
-    logger.info('Application listening on port', app.get('port'));
-    refreshCache(client,logger);
-  });
+// Initialize logger
+const log4js = require('log4js');
+log4js.configure({
+	'appenders': [
+		{
+			type: 'console',
+			category: 'purdue-laundry'
+		},
+		{
+			type: 'loggly',
+			token: '5ac059d3-1b8c-4e5e-9059-0b6b080b3fee',
```

Comment:
This probably isnt an issue because we have had this in there before, but is this meant to be secret?

Reply:
that is a great question. I'm not too familiar with the service, so I'd ask whoever created this thing. Was it @garr741